pkg/testrig/template/maindb.go

```diff
@@ -0,0 +1,66 @@
+package template
+
+func GetMainDbStub() string {
+	return `package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"database/sql"
+	"time"
+
+	"github.com/go-chi/chi"
+	_ "github.com/lib/pq"
+
+	{{name}} "{{url}}"
+	{{impl.name}} "{{impl.url}}"
+)
+
+func initDb() (*sql.DB, error) {
+	psqlInfo := "host={{name}}_db port=5432 user=someuser password=somepassword dbname=somedb sslmode=disable"
+	db, err := sql.Open("postgres", psqlInfo)
+	if err != nil {
+		return nil, err
+	}
+
+	time.Sleep(5 * time.Second)
```

Comment:
a 5 second sleep here is a bit nasty. Is it really needed? A comment would not go astray if it is needed.

Reply:
it is nasty
really what's needed here is proper knock-knock cycle
as we are not advertise this stuff yet, I'd better implement it